dts/x86/intel/alder_lake.dtsi

```diff
@@ -430,6 +430,13 @@
 			status = "okay";
 		};
 
+		tgpio: tgpio@fe001200 {
+			compatible = "intel,timeaware-gpio";
+			reg = <0xfe001200 0x100>;
+			timer-clock = <19200000>;
+			max-pins = <2>;
+			status = "okay";
```

Comment:
Nitpick: missing empty line after this

Reply:
Clarification: after the node, ie have an empty line between each node definition.

Reply:
Updated